README.md

```diff
@@ -40,6 +40,12 @@ firewall rules may not be able to contact the Charm Store. In these network
 restricted  environments the resource can be uploaded to the model by the Juju
 operator.
 
+## Deploying on LXD
+
+When deploying on LXD this charm will deploy its own profile, this profile can be inspected by running `lxd profile list` and choosing the profile matching your juju model name, once you have this you can run `lxc profile show ${PROFILE_NAME}`.
```

Comment:
Is juju going to rename this profile to match the model name?

Reply:
Yes, I will clear up docs to clarify this